src/domains/shared/components/postcard/PostCard.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from '@emotion/styled';
+import { PostCardProps } from './PostCardType';
+
+const PostCard = ({ imageSrc, title, summary, authors }: PostCardProps) => {
+  return (
+    <CardLayout>
+      <ImageSection>Image</ImageSection>
+      <DiscriptionBox>
+        <CardTitle>{title}</CardTitle>
+        <CardSummary>{summary}</CardSummary>
+      </DiscriptionBox>
+      <CardAuthor>
+        {authors.map((author) => (
+          <span>{author.name}</span>
+        ))}
+      </CardAuthor>
+    </CardLayout>
+  );
+};
+
+const CardLayout = styled('div')`
+  position: relative;
+  border-radius: 20px;
+  background-color: red;
+  width: 296px;
+  height: 331px;
+`;
```

Comment:
보통 props가 없는 경우는 이렇게 많이 쓰는거 같은데 더 코드도 적고 좋아보입니다. 어떻게 생각하세요?~

```suggestion
const CardLayout = styled.div`
  position: relative;
  border-radius: 20px;
  background-color: red;
  width: 296px;
  height: 331px;
`;
```

Reply:
좋습니다! 스트링이 없다는 것도 좋은것 같네요!